hphp/hack/test/typecheck/ctor_privacy.php.exp

```diff
@@ -1,4 +1,4 @@
 File "ctor_privacy.php", line 17, characters 14-16:
-You cannot access this member (Typing[4112])
-File "ctor_privacy.php", line 13, characters 20-30:
-This member is private
+You cannot access this property (Typing[4112])
```

Comment:
It seems wrong to call a constructor a property?

Reply:
Yep, agreed.